### PR TITLE
Upgrade PSPDFKit to 9.5.2

### PR DESCRIPTION
**Pull request by cobbal**
(This won't be able to be merged until cocoapods 1.10 is released, which should be soon)

refs: MBL-14611
affects: teacher, student
release note: none

### CODE REVIEW

**Comment by inst-danger** [comment]
## Latest Installable Builds

(will redirect to this page if no builds exist yet)
student: [Link](https://rebrand.ly/hwriyif) | [QR](https://qr.rebrandly.com/v1/qrcode?shortUrl=rebrand%2Ely%2Fhwriyif)
teacher: [Link](https://rebrand.ly/2v9ubwy) | [QR](https://qr.rebrandly.com/v1/qrcode?shortUrl=rebrand%2Ely%2F2v9ubwy)
parent: [Link](https://rebrand.ly/lqa695d) | [QR](https://qr.rebrandly.com/v1/qrcode?shortUrl=rebrand%2Ely%2Flqa695d)

**Comment by inst-danger** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>

  Build failed, skipping coverage check
  </td>
    </tr>
  </tbody>
</table>


<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  This pull request will not generate a release note.
  </td>
    </tr>
  </tbody>
</table>


[MBL-14611](https://instructure.atlassian.net/browse/MBL-14611)

#### Affected Apps: Teacher, Student
<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

**Comment by inst-danger** [comment]
<details><summary>Student.ipa</summary>
  <a href="https://app.bitrise.io/artifact/49864453/p/3d42a01b4cca9cd9e22dc4b9ed4f78c8">
    
  </a>
</details>
<details><summary>Teacher.ipa</summary>
  <a href="https://app.bitrise.io/artifact/49864455/p/849e2dd1f0e7c421aed6e0193abe8aee">
    
  </a>
</details>
<details><summary>Parent.ipa</summary>
  <a href="https://app.bitrise.io/artifact/49864433/p/194f773144659570df8a707a4c8f4ff0">
    
  </a>
</details>

File: scripts/build_automation/bitrise_workflows/tests.yml [hunk 1]
```diff
@@ -100,6 +100,7 @@ workflows:
                 -scheme CITests \
                 -sdk iphonesimulator \
                 -configuration Debug \
+                -destination 'platform=iOS Simulator,name=iPhone 8' \
```
Comment (vanwagonet) [inline scripts/build_automation/bitrise_workflows/tests.yml:103]: `iPhone SE (2nd generation)`?

File: TestsFoundation/TestsFoundation.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -1136,9 +1136,11 @@
 				PRODUCT_BUNDLE_IDENTIFIER = com.instructure.TestsFoundation;
 				PRODUCT_NAME = "$(TARGET_NAME:c99extidentifier)";
 				SKIP_INSTALL = YES;
+				SUPPORTS_MACCATALYST = NO;
 				SWIFT_OPTIMIZATION_LEVEL = "-Onone";
 				SWIFT_TREAT_WARNINGS_AS_ERRORS = NO;
 				SWIFT_VERSION = 5.0;
+				TARGETED_DEVICE_FAMILY = "1,2";
```
Comment (vanwagonet) [inline TestsFoundation/TestsFoundation.xcodeproj/project.pbxproj:1143]: I just removed these, are they necessary?

Comment (cobbal) [reply TestsFoundation/TestsFoundation.xcodeproj/project.pbxproj:1143]: No idea, I just let xcode call the shots.

**Review by vanwagonet — APPROVED** [review]
QA+1 Looks great.

**Pull request merged**